worker/image-cache.js

```diff
@@ -11,39 +11,32 @@ imageToBase64Url = function(image) {
   return canvas.toDataURL("image/png");
 };
 
-imageFetch = function(success) {
-  var imageCategories = ["buildings", "nature", "objects", "people", "technology"];
+imageFetch = function(resolutionIndex, photoID, success) {
+  resolutionIndex = parseInt(resolutionIndex) || 0;
```

Comment:
Yup, again, vestigial code. Had initially had items either pass a resolutionIndex or a static photoID. Now this isn't required.